frontend-ssr/styles/_animations.scss

```diff
@@ -1,11 +1,22 @@
 @keyframes blinkElement {
-  0% {
-    opacity: 0;
-  }
-  50% {
-    opacity: 1;
-  }
-  100% {
-    opacity: 0;
-  }
+    0% {
```

Comment:
Thank you Alex. I have checked my Workspace Settings and the configuration seems to be ok.
![image](https://user-images.githubusercontent.com/18526942/96257466-15941e00-0fc3-11eb-8188-79bc72e817b0.png)
Hope that now we are "on the same page" because I have set up those commands too.